jetty-server/src/test/java/org/eclipse/jetty/server/ProxyConnectionTest.java

```diff
@@ -151,6 +151,39 @@ public void testIPv6V2(RequestProcessor p) throws Exception
         assertThat(response, Matchers.containsString("remote=ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff:12345"));
     }
 
+    @ParameterizedTest
+    @MethodSource("requestProcessors")
+    public void testLocalV2(RequestProcessor p) throws Exception
+    {
+        String proxy =
+            // Preamble
+            "0D0A0D0A000D0A515549540A" +
+
+                // V2, LOCAL
+                "20" +
+
+                // 0x1 : AF_INET    0x1 : STREAM.
+                "11" +
+
+                // Address length is 16.
+                "0010" +
+
+                // gibberish
+                "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+            ;
+        String http = "GET /path HTTP/1.1\n" +
+            "Host: server:80\n" +
+            "Connection: close\n" +
+            "\n";
+
+        String response = p.sendRequestWaitingForResponse(TypeUtil.fromHexString(proxy), http.getBytes(StandardCharsets.US_ASCII));
```

Comment:
I'm not sure you should get back a 200 OK response for a LOCAL command.

Reply:
As discussed; the connection should be upgraded and handled normally even when the command is LOCAL.